posthog/clickhouse/migrations/0074_add_custom_and_features_property_groups.py

```diff
@@ -0,0 +1,10 @@
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.clickhouse.property_groups import sharded_events_property_groups
+
+operations = [
+    run_sql_with_exceptions(statement)
+    for statement in [
+        *sharded_events_property_groups.get_alter_table_statements("custom"),
+        *sharded_events_property_groups.get_alter_table_statements("feature_flags"),
```

Comment:
```suggestion
        *sharded_events_property_groups.get_alter_create_statements("custom"),
        *sharded_events_property_groups.get_alter_create_statements("feature_flags"),
```
I would change these methods to make clear that you are creating things since alter itself is ambiguous - creating/changing/dropping?

Reply:
Makes sense to me.